### PR TITLE
Update referrals field type

### PR DESCRIPTION
**Pull request by pettinarip**
## Description

Update the `Referrals` field type to be a TextArea.

## Related Issue

#268

### CODE REVIEW

File: src/components/forms/ProjectGrantsForm.tsx [hunk 1]
```diff
@@ -590,7 +589,8 @@ export const ProjectGrantsForm: FC = () => {
             </FormLabel>
 
             <PageText as='small' fontSize='helpText' color='brand.helpText'>
-              For example, do you have an MVP? Please describe its functionality and limitations and provide a link to it here.
+              For example, do you have an MVP? Please describe its functionality and limitations and
+              provide a link to it here.
```
Comment (nhsz) [inline src/components/forms/ProjectGrantsForm.tsx:593]: seems prettier config is formatting these?

**Review by nhsz — APPROVED** [review]

**Review by wackerow — APPROVED** [review]

File: src/components/forms/ProjectGrantsForm.tsx (hunk 1 above)
Comment (pettinarip) [reply src/components/forms/ProjectGrantsForm.tsx:593]: yea... 🤔 not sure why, I guess that for some reason my prettier didn't detect the project's config. Will double-check it soon and fix these modifications

**Review by corwintines — APPROVED** [review]

File: src/components/forms/ProjectGrantsForm.tsx (hunk 1 above)
Comment (pettinarip) [reply src/components/forms/ProjectGrantsForm.tsx:593]: This is actually correct. We have a `"printWidth": 100`. Seems that were not formatted correctly before.

Comment (nhsz) [reply src/components/forms/ProjectGrantsForm.tsx:593]: Nice.

**Pull request merged**